Misc/NEWS.d/next/Build/2022-09-14-10-38-15.gh-issue-96821.Zk2a9c.rst

```diff
@@ -0,0 +1 @@
+Explicitly mark C modules that need defined signed integer overflow.  (So that in future we can turn off the blanket `-fwrapv`, and enjoy more C compiler optimizations.)
```

Comment:
I'm not sure if the parenthesised sentence makes sense as part of a NEWS entry. Just describing the actual change should suffice.

Reply:
I wanted to give a bit of context. But I'm happy to change it.

Reply:
Please see new version.